blocks/header/header.js

```diff
@@ -24,6 +24,17 @@ async function alternateHeaders() {
     }
   }
 
+  /**
+  * Use pathArray to determine the direction of the language
+  * If the language is Arabic, Hebrew or Kuwaiti Arabic, set the direction to rtl
+  */
+  let dir = 'ltr';
+  if (pathArray[1] && (pathArray[1] === 'ar-sa' || pathArray[1] === 'he-il' || pathArray[1] === 'ar-kw')) { dir = 'rtl'; }
```

Comment:
All pages now have the metadata field "og:locale" and you can use `getMetadata('og-locale')` instead of pathArray[1]